internal/cmd/skupper/common/utils/status.go

```diff
@@ -21,3 +21,12 @@ func SiteReady(siteList *v2alpha1.SiteList) (bool, string) {
 	}
 	return false, ""
 }
+
+func SiteLinkAccessEnabled(siteList *v2alpha1.SiteList) (bool, string) {
+	for _, s := range siteList.Items {
+		if s.Spec.LinkAccess == "default" {
```

Comment:
This is not correct. The link access type can have 3 different values: [route loadbalancer default]

Reply:
correct,  sorry missed that.